src/main/scala/tilelink/Parameters.scala

```diff
@@ -276,13 +276,42 @@ class TLSlaveParameters private(
         putFull       = supportsPutFull,
         putPartial    = supportsPutPartial,
         hint          = supportsHint),
-      emits           = emits,
+      emits         = emits,
       mayDenyGet    = mayDenyGet,
       mayDenyPut    = mayDenyPut,
       alwaysGrantsT = alwaysGrantsT,
       fifoId        = fifoId)
   }
 
+  def v2copy(
+    nodePath:      Seq[BaseNode],
+    resources:     Seq[Resource],
+    setName:       String,
```

Comment:
The argument should probably just be called 'name' (but still default to the value of setName).